src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const BASE_PRICE_PER_DAY = 40;
+  let rentAmount = days * BASE_PRICE_PER_DAY;
```

Comment:
Consider using `const` for `rentAmount` since it's not being reassigned after the initial calculation. This aligns with the checklist item 3, which encourages the use of `const` over `let` to avoid unintentional reassignments.